JS exercises/arrays/matriz-multiplica.js

```diff
@@ -0,0 +1,24 @@
+const multi = (width, height) => {
+  let table = [];
+
+  for(let i = 0; i< height; i++) {
+    table[i] = []
+
+    for(let j = 0; j< height; j++) {
+      table[i][j] = i
+
+      if(i>0) {
+        table[0][j] = j
+      }
+      if(j>0) {
+        table[i][j] = i*j
+      }
```

Comment:
You are updating some cells several times, can you find a way to only set every cell just once?